bin/shared-setup

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# multi-platform compatible setup script to be shared by platform specific setup scripts
+
+gem install bundler --conservative
+bundle install
```

Comment:
I noticed  `bundle install` was not in setup but `npm install` is. Is there a reason behind this? If not I think we should add it.

Reply:
IIUC, `bundle install` is ran on the docker image since the gems are stored in non-shared folder.
`npm install` installs packages on `node-modules` folder. 
So, running `bundle install` on your own machine is an optional...

Reply:
This makes sense for the app but you need them for running some of the scripts in `bin`

Reply:
This makes sense for the app but you need them for running some of the scripts in `bin`

Reply:
I guess we can run it then.
The only thing I do not like is that this can mess up your ruby env, if you are not on a virtual environment.

BTW, by curiosity, which scripts did you notice that depend on it?

Reply:
That's a good point. The `bin/shell` script depends on some aws-sdk gems which is why I ran into the issue. I can remove that and document it instead to be safe.

Reply:
Thanks.
Seems overkill to install everything else because of this script... not sure what is the best option long term here (maybe a custom gemfile for setup?).